config/samples/osp-director_v1beta1_openstackcontrolplane_with_additional_custom_role.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: osp-director.openstack.org/v1beta1
+kind: OpenStackControlPlane
+metadata:
+  name: overcloud
+  namespace: openstack
+spec:
+  openStackClientImageURL: quay.io/openstack-k8s-operators/tripleo-deploy:latest
+  ospNetwork:
+    bridgeName: br-osp
+    name: osp
+    nodeNetworkConfigurationPolicy:
+      nodeSelector:
+        node-role.kubernetes.io/worker: ""
+      desiredState:
+        interfaces:
+        - bridge:
+            options:
+              stp:
+                enabled: false
+            port:
+            - name: enp6s0
+          description: Linux bridge with enp6s0 as a port
+          name: br-osp # must be same as bridgeName
+          state: up
+          type: linux-bridge
+  virtualMachineRoles:
+    controller:
+      roleName: Controller
+      roleCount: 0
+      cores: 2
+      memory: 20
+      diskSize: 50
+      baseImageURL: http://download.eng.brq.redhat.com/brewroot/packages/rhel-guest-image/8.3/417/images/rhel-guest-image-8.3-417.x86_64.qcow2
+      storageClass: host-nfs-storageclass
+      baseImageVolumeName: openstack-base-img #optional
+      networks:
+      - ctlplane
+    somecustomrole:
+      roleName: SomeCustomRole
+      isTripleoRole: false
```

Comment:
This is renamed now right?

Reply:
no, but we default to true, so you only have to set it on non tripleo roles.

[1] https://github.com/openstack-k8s-operators/osp-director-operator/blob/11c5c1205a83[…]d4a78e85152bd0f8389d/api/v1beta1/openstackcontrolplane_types.go